### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by dili021**
Created a heatmap of the smashing magazine website with different shades of gray representing visual impact.

### CODE REVIEW

**Review by nikasulo — CHANGES_REQUESTED** [review]
# This is great work team! @dili021 and @Luckyaremu :rocket: 
Nice use of different shades of grey, as well as labeling each div. Please consider my comments below, make appropriate changes and request another review! :handshake: 

#### Horizontal Scroll
You seem to have a little bug with one of the elements of the page, shooting out beyond the browser's maximum width. This may be as a result of a margin or padding you have set up. One of the easiest ways I have found to correct this issue is to set the default `box-sizing` property to `border-box` as follows `*{box-sizing: border-box}`. Please read some more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing).

#### Using the article tag
Please consider using the `<article>` tag where appropriate, in this instance, all the articles on the page should be wrapped in this tag as opposed to using `div` tags [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png)

#### Footer with centered links
Please consider adding centered links to your footer like on the original page. The content of the links don't matter much, only that they're centered

### Summary
Your work is commendable, I like your approach! Please consider my comments above, treat them as hard requirements because they are. I wish you all the best on your next submission and the remainder of your projects! See you at the very top :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by euqueme** [comment]
### Pull Request under the review by Maria Eugenia Queme P. (TSE)

Hello Team! :wave:  you'll see the requested changes within the hour

File: README.md [hunk 1]
```diff
@@ -1,25 +1,85 @@
-# Creators
+> Design-teardown
 
+![screenshot](img/Screenshot.jpg)
 
-# What is this
+A clone of the beautifull new york times editorial.
```
Comment (euqueme) [inline README.md]: Change this line as this is a different project

File: README.md [hunk 2]
```diff
@@ -1,25 +1,85 @@
...
+## Live Demo
 
-# Tools used
+[Live Demo Link](https://raw.githack.com/dili021/Design-teardown/feature/index.html)
 
+## Getting Started
 
-# Live demo
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (euqueme) [inline README.md]: remove or modify these lines to be coherent with your project

File: style.css [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  background-color: rgb(160, 148, 148);
+}
+
+/* navbar */
+.navbar-ul {
+  list-style: none;
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+
+  /* width: 100%; */
```
Comment (euqueme) [inline style.css]: remove commented code on line #14

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Great job @dili021 and @Luckyaremu :clap: 

### Thank you for doing the changes required by the previous reviewer, there are still a couple of things to address before we can give you the approval:

- [ ] You are missing the **Playing with Typography** section from the Odin project:

all of your main articles should have meaningless text and the background color should be white as the font type, size and color would be in charge of the weight of those sections.  

image

- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history

- [ ] Your Readme file needs to be modified, please address the comments I have left in your code :arrow_double_down: 

[OPTIONAL] update your Live Demo after the changes.

### That's it, team! :raised_hands: 

### Keep up the good work! :hammer:  please :pray: make these changes :memo:  and submit back for a code review when you are done :rocket: 

Any questions? reach to me on slack **@ Maru**

Happy Coding! :woman_technologist: 

PROJECT_STATUS: REQUESTED_CHANGES :traffic_light:

File: index.html [hunk 4]
```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>design-teardown</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
```
Comment (euqueme) [inline index.html]: Add the [link to  **Zhi Mang Xing** font](https://fonts.google.com/?query=Zhi+Mang+Xing&selection.family=Zhi+Mang+Xing) after the last link in your **\<head\>** tag

` <link href="https://fonts.googleapis.com/css?family=Zhi+Mang+Xing&display=swap" rel="stylesheet">`

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello again Team! :handshake: 

### Thank you for doing most of my changes we still have a couple of mandatory ones and you are good to go:

- [ ] I see you're using the **Zhi Mang Xing font** in your CSS file but you didn't import in your HTML file, please look at the comment I have done to your code to import it correctly in your HTML file. :arrow_double_down: 

- [ ] The background for the articles that have text have to be white as stated in the Odin project:

image

##
### That is it Team :ok_woman:

[OPTIONAL] don't forget to update your Live Demo once the changes are done

### Please make the changes :raised_hands:  and submit back for a code review when you are done :information_desk_person:

Happy Coding! :woman_technologist: 

Any questions? reach to me on slack **@ Maru**

PROJECT_STATUS : REQUESTED CHANGES :vertical_traffic_light:

**Review by MauricioRobayo — APPROVED** [review]
Very nice project!

Love the use of CSS transitions for the hover effects, well done 👍.

Also nice that you setup Netlify for the live demo. For this kind of project [gh-pages](https://pages.github.com/) is also a nice option, but kudos for setting up Netlify 🥇.

Please don't forget to edit this PR title and description, that's a requirement:

* [ ] Pull Request has meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

I am confident that you'll do that before you merge this. Keep up the great work 💪.

Status: **Approve**.

**[OPTIONAL]**: You are doing great using semantic HTML, you could also use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) tag to wrap the dominant content of the `<body>` of the document.

**Pull request merged**